ocaml/bytecomp/bytegen.ml

```diff
@@ -973,7 +973,7 @@ let rec comp_expr stack_info env exp sz cont =
         | CFnge -> Kccall("caml_ge_float", 2) :: Kboolnot :: cont
       in
       comp_args stack_info env args sz cont
-  | Lprim (Pfloatcomp (Pfloat32, cmp), args, _) ->
+  | Lprim (Pfloatcomp (Pfloat32, cmp), args, _) | Lprim (Punboxed_float_comp (Pfloat32, cmp), args, _) ->
```

Comment:
Is the new case reachable?  I think we would need to have literal patterns for unboxed float32s, and we don't have either.

Reply:
It doesn't look like this is specific to pattern matching, so it may be reachable if the middle end generates an unboxed float comp. In bytecode mode I imagine it won't, but `float32#` is coming soon anyway.